### PR TITLE
Proxy conf for Leantime

### PR DESCRIPTION
**Pull request by mentallyinspired**
Proxy configuration for the project management tool Leantime.
https://leantime.io

[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]

### CODE REVIEW

File: leantime.subdomain.conf.sample [hunk 1]
```diff
@@ -0,0 +1,40 @@
+## Version 2022/08/06
```
Comment (Roxedus) [inline leantime.subdomain.conf.sample:1]: This should also mention the need for a CNAME

**Review by Roxedus — CHANGES_REQUESTED** [review]

**Review by nemchik — CHANGES_REQUESTED** [review]
The proxy templates have been updated with new descriptive comments and auth lines. Please update your PR accordingly.

Ref:
https://github.com/linuxserver/reverse-proxy-confs/blob/master/_template.subdomain.conf.sample
https://github.com/linuxserver/reverse-proxy-confs/blob/master/_template.subfolder.conf.sample

**Review by mentallyinspired — COMMENTED** [review]
DNS cname info and container name info text

File: leantime.subdomain.conf.sample [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+    include /config/nginx/ssl.conf;
+
+    client_max_body_size 20M;
+
+    # enable for ldap auth, fill in ldap details in ldap.conf
+    #include /config/nginx/ldap.conf;
+
+    # enable for Authelia
+    #include /config/nginx/authelia-server.conf;
```
Comment (nemchik) [inline leantime.subdomain.conf.sample:19]:
```suggestion
    client_max_body_size 0;

    # enable for ldap auth (requires ldap-location.conf in the location block)
    #include /config/nginx/ldap-server.conf;

    # enable for Authelia (requires authelia-location.conf in the location block)
    #include /config/nginx/authelia-server.conf;

    # enable for Authentik (requires authentik-location.conf in the location block)
    #include /config/nginx/authentik-server.conf;
```

File: leantime.subdomain.conf.sample [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        # enable the next two lines for http auth
+        #auth_basic "Restricted";
+        #auth_basic_user_file /config/nginx/.htpasswd;
+
+        # enable the next two lines for ldap auth
+        #auth_request /auth;
+        #error_page 401 =200 /ldaplogin;
+
+        # enable for Authelia
+        #include /config/nginx/authelia-location.conf;
```
Comment (nemchik) [inline leantime.subdomain.conf.sample:31]:
```suggestion
        # enable for ldap auth (requires ldap-server.conf in the server block)
        #include /config/nginx/ldap-location.conf;

        # enable for Authelia (requires authelia-server.conf in the server block)
        #include /config/nginx/authelia-location.conf;

        # enable for Authentik (requires authentik-server.conf in the server block)
        #include /config/nginx/authentik-location.conf;
```

**Review by nemchik — CHANGES_REQUESTED** [review]
Might seem real picky, but having all the comments consistent makes it easier for end users to use and easier for us to maintain (especially when we have to do mass find/replace).

File: leantime.subdomain.conf.sample (hunk 3 above)
Comment (nemchik) [reply leantime.subdomain.conf.sample:34]: Need this one too.

**Review by nemchik — APPROVED** [review]
lgtm. thanks!

**Pull request merged**